test/test_task_on_kart.py

```diff
@@ -657,5 +658,18 @@ def test_run_when_complete_check_at_run_is_true_and_task_is_completed(self):
         task.dump.assert_not_called()
 
 
+class TestPickleTaskOnKart:
+    def test_pickle_and_unpickle(self):
+        task = _DummyTask(redis_host='0.0.0.0', redis_port=12345, redis_timeout=180, should_lock_run=True)
+        pickled = pickle.dumps(task)
```

Comment:
@hiro-o918 
Since gokart uses `dill.dumps()`, I thought this may not happen in runtime.

If this test is changed to dill.dumps(), would it solve the problem that motivated this pull request?

Reply:
I've understood. This was to reproduce the effect of following code.

```
gokart.build(_ParallelRunner(), reset_register=False, workers=20)
```